yarn-project/ethereum/src/utils.ts

```diff
@@ -64,3 +66,110 @@ function tryExtractEvent<
     }
   }
 }
+
+export function prettyLogViemErrorMsg(err: any) {
+  if (err instanceof BaseError) {
+    const revertError = err.walk(err => err instanceof ContractFunctionRevertedError);
+    if (revertError instanceof ContractFunctionRevertedError) {
+      const errorName = revertError.data?.errorName ?? '';
+      const args =
+        revertError.metaMessages && revertError.metaMessages?.length > 1 ? revertError.metaMessages[1].trimStart() : '';
+      return `${errorName}${args}`;
+    }
+  }
+  return err?.message ?? err;
+}
```

Comment:
The idea would be to drop this function in favor of the other? Or to keep both?

Reply:
yeah I think this isn't used anywhere anymore actually, can remove